src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -43,8 +49,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String userEmail = userService.getCurrentUser().getEmail();
-    System.out.println("Saving about me for " + userEmail);
-    // TODO(kcastaneda): Save the data
+    String aboutMe = Jsoup.clean(request.getParameter("about-me"), Whitelist.none());
```

Comment:
Put this clean above the declaration of userEmail since we usually want to clean first then get whatever data we need after.